### PR TITLE
Masonry: Fix integration tests

### PR DESCRIPTION
**Pull request by JosAnGoCa**
### Summary

Mansory: _Patch_ the execution of the tests.

#### What changed?

This pr includes the fix for the error `Error: page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL` while executing the mansonry integration tests
image

Also added manual retries for the flaky test: `playwright/masonry/null-items.spec.ts`

#### Why?

To help in the execution of the mansory tests.

### Checklist

- [x] Added a manually retry to a test

### CODE REVIEW

File: playwright/masonry/utils/getServerURL.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const BASE_URL = 'http://localhost:8888';
```
Comment (IsraLopez26) [inline playwright/masonry/utils/getServerURL.ts:1]: @JosAnGoCa  apparently this would only work in local environment. Can we please remove this line and see if the tests pass now?

**Review by AlbertCarreras — APPROVED** [review]
